packages/story-editor/src/output/utils/styles.js

```diff
@@ -136,6 +136,27 @@ function CustomStyles() {
                   transform: var(--initial-transform);
                 }
               }
+
+              .captions-area {
+                width: 100%;
+                height: 100%;
+                display: grid;
+                align-content: end;
+              }
+
+              .captions-scrim {
+                padding: 32px;
+                background: linear-gradient(to bottom, transparent, rgba(0, 0, 0, 0.8));
+                color: rgba(255, 255, 255, 1);
```

Comment:
Nit: `color: white`?

Reply:
Yeah maybe. Didn't bother fleshing this out since styling needs refinement anyway.